lib/datasource/docker/index.ts

```diff
@@ -535,3 +543,35 @@ export async function getPkgReleases({
   }
   return ret;
 }
+
+function getECRAuthToken(region: string, opts: hostRules.HostRule) {
+  const config = { region, accessKeyId: undefined, secretAccessKey: undefined };
+  if (opts.username && opts.password) {
+    config.accessKeyId = opts.username;
+    config.secretAccessKey = opts.password;
+  }
+  const ecr = new AWS.ECR(config);
+  return new Promise<string>((resolve, reject) => {
+    ecr.getAuthorizationToken({}, (err, data) => {
+      if (err) {
+        logger.error({ err });
```

Comment:
```suggestion
        logger.warn({ err });
```

I reserve `logger.error` for things that are wrong with the bot itself. User config errors should be info or warn.